tests/hellinger_distance/test_estimated_hellinger_distance.py

```
@@ -152,40 +165,81 @@ def test_hellinger_distance_error() -> None:
         hellinger_distance(p=invalid, q=valid)
 
 
-def test_train_random_forest_regressor_and_predict(device: Target) -> None:
-    """Test the training of the random forest regressor. The trained model is saved and used in the following tests."""
-    # Setup the training environment
+@pytest.mark.parametrize(("model_type", "verbose"), [("rf", False), ("gnn", False), ("gnn", True)], ids=["rf", "gnn", "gnn_verbose"])
+def test_train_model_and_predict(device: Target, model_type: str, verbose: bool) -> None:
```

Ruff (0.14.10)

Boolean-typed positional argument in function definition

(FBT001)